### DISCUSSION TITLE
Problem 1     - phpunit/phpunit[10.1.0, ..., 10.3.2] require sebastian/cli-parser

### DISCUSSION THREAD

**Discussion by huaiguoguo** [laravel/framework#48270] [Q&A]
### Laravel Version

latest

### PHP Version

8.2.30

### Database Driver & Version

mysql 8.0.31

### Description

i am accident some problem:

![image](https://github.com/laravel/framework/assets/1888094/0e329e7b-4b74-4157-bec9-11cf9824c3ae)

### Steps To Reproduce

laravel new laravel_demo
composer create-project laravel/laravel  laravel_test

accent this problem
![image](https://github.com/laravel/framework/assets/1888094/0e7a0afa-01a4-40d1-93d7-13aa352a0352)

**Comment by henzeb** [comment]
works on my machine... get the same laravel version. try composer clear-cache

**Comment by huaiguoguo** [comment]
> works on my machine... get the same laravel version. try composer clear-cache

i found way to fixed problem
it was work!

**Comment by tennessine** [comment]
> 
怎么解决的？在debian 11 php 8.1上安装不了，现在用laravel这么复杂了吗

**Reply by huaiguoguo** [reply]
> > 
> 怎么解决的？在debian 11 php 8.1上安装不了，现在用laravel这么复杂了吗
> 

我的是因为镜像源的问题，换一下镜像源，重新执行，好了
